### ISSUE TITLE
Jupyter application fails to start when Service Mesh is enabled

### ISSUE THREAD

**Issue by bartoszmajsak** [opendatahub-io/opendatahub-operator#616]
**Describe the bug**

Code in #605 

Notebook pods created in the Application Namespace (so the same as controllers) have a permission issue resulting in `CrashLoopBackOff` of the Jupyter container (which has `istio-proxy` sidecar injected):

```
PermissionError: [Errno 13] Permission denied: '/opt/app-root/src/.local'
```
 It does work however when the same Jupyter workbench is created in a new namespace (Data Science Project).

**To Reproduce**

- go to ODH Dash Dashboard
- select applications
- enable jupyter nb (any kind should result in the same)

**Expected behavior**
Application is created and accessible

**Additional context**

Here are the pod details of both scenarios:

- workbench in its own ns: https://gist.github.com/bartoszmajsak/5b12ffd4301d7326fd2a704c9481381a
- workbench in app namespace (opendatahub): https://gist.github.com/bartoszmajsak/a29a39b19a4b5690c029c90262d25a82

**WORKAROUND** 

Use SMCP explicitly set to `v2.3` to get this working.

**Comment by zdtsw** [comment]
this bug is actually not in the "incubation" branch yet, right?
it will be a bug once pr 605 gets merged in.

**Comment by AjayJagan** [comment]
here is the JIRA counterpart for this issue: https://issues.redhat.com/browse/RHOAIENG-406

**Comment by bartoszmajsak** [comment]
> this bug is actually not in the "incubation" branch yet, right? it will be a bug once pr 605 gets merged in.

yes, and it's fixed in https://github.com/opendatahub-io/opendatahub-operator/pull/605/commits/0bd6ab9230d713799f82b5a6616b57fd05a979ec

**Comment by AjayJagan** [comment]
> > this bug is actually not in the "incubation" branch yet, right? it will be a bug once pr 605 gets merged in.
> 
> 
> 
> yes, and it's fixed in https://github.com/opendatahub-io/opendatahub-operator/pull/605/commits/0bd6ab9230d713799f82b5a6616b57fd05a979ec

So we close it here and in the JIRA -- wdyt?

**Comment by zdtsw** [comment]
close github issue keep the one in JIRA till code gets merged
